torch/csrc/jit/passes/frozen_conv_add_relu_fusion_impl.cpp

```diff
@@ -0,0 +1,127 @@
+#include <ATen/Utils.h>
+
+#include <torch/csrc/jit/frontend/code_template.h>
+#include <torch/csrc/jit/ir/constants.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/ir/subgraph_matcher.h>
+#include <torch/csrc/jit/passes/frozen_conv_add_relu_fusion.h>
+#include <torch/csrc/jit/passes/graph_rewrite_helper.h>
+#include <torch/csrc/jit/passes/remove_mutation.h>
+#include <torch/csrc/jit/passes/subgraph_rewrite.h>
+#ifdef USE_CUDA
```

Comment:
You don't really need `USE_CUDA` here, as this file should not even be compiled for the CPU, shouldn't it?

Reply:
In the internal build I can see why that is true - but how does that work in the OSS build?